### PR TITLE
LPS-117113 Add SidebarPanelInfoView test

### PR DESCRIPTION
**Pull request by dgarciasarai**
**Description**

Add jest test for SidebarPanelInfoView

cc: @carloslancha

### CODE REVIEW

**Comment by dgarciasarai** [comment]
ci:test:sf

**Comment by dgarciasarai** [comment]
ci:test:relevant

**Comment by dgarciasarai** [comment]
ci:stop

File: modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/SidebarPanelInfoView.js [hunk 1]
```diff
@@ -90,8 +90,8 @@ const SidebarPanelInfoView = ({
 
 					<p className="component-subtitle">{subType}</p>
 
-					{versions.map((version) => (
-						<div key={version}>
+					{versions.map((version, index) => (
+						<div key={index}>
```
Comment (carloslancha) [inline modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/SidebarPanelInfoView.js:94]: `version.version` should be unique, isn't it?

Comment (dgarciasarai) [reply modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/SidebarPanelInfoView.js:94]: Yes, I'll change it 😄

**Comment by dgarciasarai** [comment]
ci:test:sf

**Comment by dgarciasarai** [comment]
ci:test:relevant

**Comment by carloslancha** [comment]
image
image
image

😮 😮 😮 😮 😮 😮 😮 😮

**Comment by dgarciasarai** [comment]
@carloslancha that was fast...and weird xD

**Comment by carloslancha** [comment]
This looks like a test that could be covered by poshi to me.

What I'd expect for [LPS-116501](https://issues.liferay.com/browse/LPS-116501) is a couple of tests checking the sidebar open/close behavior.

Now that this is done I'd keep it but under [LPS-117113](https://issues.liferay.com/browse/LPS-117113), and maybe adding a snapshot check.

What do you think?

**Comment by dgarciasarai** [comment]
ci:stop

**Comment by dgarciasarai** [comment]
You're right.
I'm going to change the LPS of the commits to LPS-117113. I'll try to do the open/close behavior under LPS-116501.
Why do you want a snapshot check?

Thanks for the feedback :)

**Comment by carloslancha** [comment]
> Why do you want a snapshot check?

Just to add the markup to the test so if something changes in the markup that could cause a visual break we are forced, at least, to be sure that what we changed was intended (if markup changes snapshot check fails unless you update it)

It's only a suggestion, you decide ;)

**Comment by dgarciasarai** [comment]
Sure! Continues in https://github.com/liferay-frontend/liferay-portal/pull/177.

**Pull request closed without merging**